src/nevermined/Assets.ts

```diff
@@ -190,6 +190,18 @@ export class Assets extends Instantiable {
                 )
             )
 
+            this.logger.debug('NTF721 Aave Credit Template')
+            const nftAaveCreditServiceAgreementTemplate = await templates.aaveCreditTemplate.getServiceAgreementTemplate()
+
+            await ddo.addService(
```

Comment:
This service will be added by default when creating an NFT721. Is that the expected behavior?

Reply:
Ideally the services are specified to the create function but the current interface in `assets.createNft721` does not support this.